plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtOvsCreateTunnelCommandWrapper.java

```diff
@@ -40,14 +40,11 @@ public Answer execute(final OvsCreateTunnelCommand command, final LibvirtComputi
         try {
             if (!libvirtComputingResource.findOrCreateTunnelNetwork(bridge)) {
                 s_logger.debug("Error during bridge setup");
-                return new OvsCreateTunnelAnswer(command, false,
-                        "Cannot create network", bridge);
+                return new OvsCreateTunnelAnswer(command, false, "Cannot create network", bridge);
             }
 
-            libvirtComputingResource.configureTunnelNetwork(command.getNetworkId(), command.getFrom(),
-                    command.getNetworkName());
-
-            final Script scriptCommand = new Script(libvirtComputingResource.getOvsTunnelPath(), libvirtComputingResource.getTimeout(), s_logger);
+            libvirtComputingResource.configureTunnelNetwork(command.getNetworkId(), command.getFrom(), command.getNetworkName());
+            final Script scriptCommand = new Script(libvirtComputingResource.getOvsTunnelPath(), libvirtComputingResource.getTimeout().getMillis(), s_logger);
```

Comment:
Scripts is an external package, hope they will upgrade to Duration after Java 1.8.